.github/workflows/build-publish-springbatch-football-image.yml

```diff
@@ -0,0 +1,72 @@
+# This workflow will clone and build another Java project with Maven, and build and push Docker images to the GitHubs container registry
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+# https://docs.github.com/actions/use-cases-and-examples/publishing-packages/publishing-docker-images
+
+name: Publish SpringBatch Football images
+
+on:
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+    working-directory: spring-batch-football
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_PREFIX: ${{ github.repository_owner }}
+  IMAGE_NAME: spring-batch-football
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+      attestations: write
+      id-token: write
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up JDK 17
+        uses: actions/setup-java@v4
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+          cache: maven
+      - name: Clone source repo
```

Comment:
we can ignore this step if we just need to clone this repo instead of another one, because line29 does this same thing.